src/Components/PokemonCard.js

```diff
@@ -0,0 +1,39 @@
+import React, {useState} from 'react';
+import axios from "axios";
+import {useEffect} from "react";
+
+function PokemonCard({name,classname}) {
+
+    const [pokemonData, setPokemonData] = useState({})
```

Comment:
Ik zou de default waarde aanpassen naar undefined, zodat je een duidelijker verschil hebt tussen of het al wél of niet is ingeladen
```suggestion
    const [pokemonData, setPokemonData] = useState();
```